ci/docker/linux-apt-docs.dockerfile

```diff
@@ -74,13 +74,7 @@ RUN wget -q -O - https://deb.nodesource.com/setup_${node}.x | bash - && \
     rm -rf /var/lib/apt/lists/* && \
     npm install -g yarn
 
```

Comment:
Need to add the copy statement so the image layer has the requirements file at build time:

```
COPY docs/requirements.txt /arrow/docs/
RUN pip install -r docs/requirements.txt meson
```

Reply:
Also add `!docs/requirements*.txt` after this line https://github.com/apache/arrow/blob/master/.dockerignore#L30 in order to add the requirements file to the `docker context` where the COPY statement above gets it.

We need to ignore most of the file in the .dockerignore to prevent docker cache invalidation in certain circumstances.

Reply:
Then run `archery docker run conda-python-docs` :)

Reply:
Will do!
Thanks for additional instructions, I was lost =)